### ISSUE TITLE
Optimization fails on larger case study (heating case)

### ISSUE THREAD

**Issue by daren-thomas** [architecture-building-systems/CityEnergyAnalyst#2533]
The following case (based in Zurich) fails to run through with the workflow
[kleinalbis_workflow.zip](https://github.com/architecture-building-systems/CityEnergyAnalyst/files/4098146/kleinalbis_workflow.zip)

Steps to reproduce:
- download and extract project/scenario: https://polybox.ethz.ch/index.php/s/MqDIv3M2RQkNbPA
- run optimization with the following config:
```
λ cea-config optimization
City Energy Analyst version 2.28.1
Configuring `cea optimization` with the following parameters:
- general:scenario = C:\Users\cea\Documents\CityEnergyAnalyst\projects\FGZ\kleinalbis
  (default: {general:project}\{general:scenario-name})
- general:multiprocessing = False
  (default: True)
- general:number-of-cpus-to-keep-free = 1
  (default: 1)
- optimization:district-heating-network = True
  (default: True)
- optimization:district-cooling-network = False
  (default: False)
- optimization:detailed-electricity-pricing = False
  (default: False)
- optimization:population-size = 5
  (default: None)
- optimization:number-of-generations = 2
  (default: 100)
- optimization:random-seed = 100
  (default: 100)
- optimization:crossover-prob = 0.9
  (default: 0.9)
- optimization:mutation-prob = 0.1
  (default: 0.1)
```

Expected: Optimization runs through.

Buggy behavior: See following output:

```
B1147
Traceback (most recent call last):
  File "C:\Users\cea\Documents\CityEnergyAnalyst\Dependencies\Python\Scripts\cea-script.py", line 11, in <module>
    load_entry_point('cityenergyanalyst', 'console_scripts', 'cea')()
  File "c:\users\cea\documents\github\cityenergyanalyst\cea\interfaces\cli\cli.py", line 66, in main
    script_module.main(config)
  File "c:\users\cea\documents\github\cityenergyanalyst\cea\optimization\optimization_main.py", line 117, in main
    moo_optimization(locator=locator, weather_file=weather_file, config=config)
  File "c:\users\cea\documents\github\cityenergyanalyst\cea\optimization\optimization_main.py", line 83, in moo_optimization
    weather_file, district_heating_network, district_cooling_network)
  File "c:\users\cea\documents\github\cityenergyanalyst\cea\optimization\preprocessing\preprocessing_main.py", line 64, in preproccessing
    DHN_barcode=DHN_barcode)
  File "c:\users\cea\documents\github\cityenergyanalyst\cea\technologies\substation.py", line 53, in substation_main_heating
    float_format='%.3f')
  File "C:\Users\cea\Documents\CityEnergyAnalyst\Dependencies\Python\lib\site-packages\pandas\core\frame.py", line 1745, in to_csv
    formatter.save()
  File "C:\Users\cea\Documents\CityEnergyAnalyst\Dependencies\Python\lib\site-packages\pandas\io\formats\csvs.py", line 156, in save
    compression=self.compression)
  File "C:\Users\cea\Documents\CityEnergyAnalyst\Dependencies\Python\lib\site-packages\pandas\io\common.py", line 397, in _get_handle
    f = open(path_or_buf, mode)
IOError: [Errno 2] No such file or directory: 'C:\\Users\\cea\\Documents\\CityEnergyAnalyst\\projects\\FGZ\\kleinalbis\\outputs\\data\\optimization\\substations\\Total_DH_1111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111.csv'
```

**Comment by daren-thomas** [comment]
@JIMENOFONSECA it really _does_ seem to be an issue with the path length - moving the project to `C:\` solved it.

This is somewhat annoying - but I'm not sure it's really our mistake: There is no sane way to ensure, that paths are _never_ too long, as the user can always create a deeply nested project directory.

We _could_ think of a way to map the `Total_DH_xyz.csv` file to a shorter name, e.g. use the id of the individual? and keep a master list of individuals in the `optimization` folder - with a population only needing to reference which individual is being used. something like that?

**Comment by JIMENOFONSECA** [comment]
solved in #2553
